build/circle_setup.sh

```diff
@@ -7,9 +7,9 @@ set -e
 if [[ ! -d ${HOME}/scratch/data/aa_conn ]]; then
     # Folder for downloads
     mkdir -p ${HOME}/downloads
-    wget -c -O ${HOME}/downloads/AA_Conn.tar "http://googledrive.com/host/0BxI12kyv2olZY0ZkZVB4TUVtSWM"
+    wget -c -O ${HOME}/downloads/testdata.tar "https://3552243d5be815c1b09152da6525cb8fe7b900a6.googledrive.com/host/0BxI12kyv2olZVUswazA3NkFvOXM/ds054_downsampled.tar.gz"
```

Comment:
The new file is compressed. Also, you can remove the ``` -O ${HOME}/downloads/testdata.tar``` because the link I provided has filename (it didn't before, that's why there is that ```-O``` argument.